### ISSUE TITLE
add parameter hide ability in setting

### ISSUE THREAD

**Issue by MingboPeng** [ladybug-tools/design_explorer_lite#10]

**Comment by paul42** [comment]
Do you think this would be just another field in settings.json -> parameters.fieldname.hide, and if true, just don't make the slider for that one?

**Comment by MingboPeng** [comment]
Yes. and most likely I would hide some output parameters, which we usually record for self QC, not for client.

**Comment by chriswmackey** [comment]
Makes sense.  We already have a means of not displaying certain outputs so having the ability to not show certain inputs also seems prudent.
